docs/rst/tutorials/cloud/microservices_cloud/microservices_cloud.rst

```diff
@@ -0,0 +1,353 @@
+          Command: ``ros2 run microservers_py server_addition``
+
+    * **subtraction_service**
+
+        * **Client**: generate 2 random numbers from 0 to 100 and send request to server.
+          Command: ``ros2 run clients_py client_subtraction``
+
+        * **Server**: subtract 2 integers get in request, and send solution in response.
+          Command: ``ros2 run microservers_py server_subtraction``
+
+This files could be found in this same documentation in ``code/microservices`` as a Colcon python package.
+In order to install these applications, follow these steps:
+
+Start by creating the workspace of this application and downloading the source code:
+
+.. code-block:: bash
+
+    mkdir -p clients_ws/src && cd clients_ws/src
+    git clone --branch humble https://github.com/eProsima/vulcanexus.git
+    mv vulcanexus/code/microservices/clients_py .
+    rm -rf vulcanexus
+    cd ..
+
+Once created the workspace, source the Vulcanexus environment and build the ``clients_py`` package.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    colcon build
+
+Then source the ``clients_ws`` workspace:
+
+.. code-block:: bash
+
+    source clients_ws/install/setup.bash
+
+And finally, run the application:
+
+.. code-block:: bash
+
+    ros2 run clients_py client_addition # client_subtraction
+
+.. note::
+
+    Since the purpose of this tutorial is not to explain how to create a ROS 2 node, but rather, the communication of these in distributed environments, the code will not be discussed in detail.
+    This code has been developed following this `ROS 2 tutorial <https://docs.ros.org/en/foxy/Tutorials/Beginner-Client-Libraries/Writing-A-Simple-Py-Service-And-Client.html>`_
+    and the comments along the code make it self-explanatory.
+    Stay tuned for new tutorials in which we will discuss how to configure ROS 2 nodes, publishers/subscribers, services and actions.
+
+Running DDS Router Edge
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Then, to run the DDS Router Edge configure the environment as in the previous step.
+
+.. note::
+
+    If you are deploying Vulcanexus from the Docker image, note that you will need to have a configuration file (``config.yaml``) to configure the DDS Router Edge accessible from your Docker container.
+
+    You can do this by mounting a shared volume when launching the container, by copying the file from the local host to the container in case it is already running, or by editing a file from the Docker container itself.
+
+Setup the Vulcanexus environment, either in a Docker container or on the local host, running the following command:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+
+Let's create the DDS Router configuration file. It will look like the one shown below.
+
+.. literalinclude:: /resources/tutorials/cloud/microservices_cloud/dds_router_edge.yaml
+    :language: yaml
+
+Next, it is briefly explained the most relevant aspects of this configuration file.
```

Comment:
p

Reply:
check if mistake also in other tutorials please.

Reply:
:+1: